### ISSUE TITLE
[Bug]: Channel page tab element highlighted on keypress

### ISSUE THREAD

**Issue by efb4f5ff-1298-471a-8973-3d47447115dc** [FreeTubeApp/FreeTube#4080]
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the bug report I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [X] This issue contains only one bug.

### Describe the bug

Case 1. Go to a channel page -> scroll down using arrow key -> page scrolls down 
Case 2. Go to a channel page -> Click on one of the tabs within the channel page (e.g. Shorts) -> scroll down using arrow key -> page scrolls down and Shorts element is highlighted

https://github.com/FreeTubeApp/FreeTube/assets/73130443/f0a48b27-0a5e-4c4c-be1a-0ae1a9b81cc9

### Expected Behavior

Element should not be highlighted when i press the arrow key

### Issue Labels

inconsistent behavior

### FreeTube Version

v0.19.0-nightly-3505 Beta

### Operating System Version

Win10 22H2

### Installation Method

.exe

### Primary API used

Local API

### Nightly Build

- [X] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Update: Element gets highlighted when u press any key

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Noticed that its not even related to the arrow key but every key.

1. Go to channel page
2. press random key
3. see that the video tab is highlighted

**Comment by msagr** [comment]
@efb4f5ff-1298-471a-8973-3d47447115dc , are we planning to remove yellow outline visual indicator entirely as we have grey ```border-block-end``` for its representation ?

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
No. I think you dont fully understand. Please try to reproduce yourself with the steps above and you will see that it will randomly highlight when it isnt supposed to highlight

**Comment by Shadorc** [comment]
This concerns all keys, not just up and down arrows. This is kind of a similar issue to #9119: clicking on an element gives it `:focus` and pressing a key will give it `:focus-active`, leading to outlines.

This seems to be an accessibility feature and the default behaviour in any browser (at least Chromium-based). You can test on GitHub (and Twitch too): click the Issues tab, and then press an arrow key (or any key, but it could trigger another shortcut leading to focus loss), and you will obtain the same behaviour.

YouTube does not have this behaviour, but a weird one instead. If you reproduce in YouTube by clicking and then pressing an arrow key, it seems to reload the page.

I agree this is a weird behaviour, but it seems to be the expected one.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Alright ill close it. Thanks for looking into this one!
